documentation/sdk/docs/how_tos/accounts_and_addresses/list_transactions.mdx

```diff
@@ -19,6 +19,14 @@ import PythonCode from "!!raw-loader!../../../../../bindings/python/examples/how
 
 This How-To will show you how to list all transactions of an account.
 
+:::tip Sync Options
+
+If you want to list incomming or pending transactions you need to specify that in your `sync` call. 
```

Comment:
For pending transactions you don't need to specify anything

Reply:
```suggestion
If you want to list incoming transactions you need to specify that in your `sync` call. 
```